docs/development.rst

```diff
@@ -41,29 +41,14 @@ Create container desired version of ``clickhouse-server``:
 
         docker run --rm -p 127.0.0.1:9000:9000 -p 127.0.0.1:8123:8123 --name test-clickhouse-server clickhouse/clickhouse-server:$VERSION
 
-Create container with the same version of ``clickhouse-client``:
+Or run the docker-compose defined in tests folder:
 
     .. code-block:: bash
 
-        docker run --rm --entrypoint "/bin/sh" --name test-clickhouse-client --link test-clickhouse-server:clickhouse-server clickhouse/clickhouse-client:$VERSION -c 'while :; do sleep 1; done'
+        cd tests && docker compose up -d
 
-Create ``clickhouse-client`` script on your host machine:
```

Comment:
This section was dedicated for running tests on machine without `clickhouse-client` installed.

is it still possible to run tests on such machine?